src/ucp/wireup/wireup_ep.c

```diff
@@ -10,6 +10,7 @@
 
 #include "wireup_ep.h"
 #include "wireup.h"
+#include "wireup_cm.h"
```

Comment:
can we remove this include? and now all changes in wireup_ep.c are not really needed for this PR right?